p2p/src/authenticated/actors/tracker/actor.rs

```diff
@@ -27,7 +27,11 @@ use std::{
 };
 use tracing::{debug, trace};
 
-const NAMESPACE: &[u8] = b"_COMMONWARE_P2P_AUTHENTICATED_IP_";
+const NAMESPACE_SUFFIX_IP: &[u8] = b"_IP";
+
+fn suffix_namespace(namespace: &[u8]) -> Vec<u8> {
```

Comment:
I wonder if we should just use `union` directly (given this is just a single line function)? I know I did this but looking at it again, seems like extra complexity. Curious if you have thoughts.